## Abstract

Phylogeographic studies have often relied on surveying neutral genetic variation in natural population as a way of gaining better insights into the evolutionary processes shaping present day population demography. Recent emphasis on understanding putative adaptive variation have brought to light the role of epigenetic variation in influencing phenotypes and local adaptation. While much is known about how methylation acts at specific loci to influence known phenotypes, there is little information on the spatial genetic structure of genome-wide patterns of methylation and the extent to which it can extend our understanding of both neutral and putatively adaptive processes.  This research examines spatial genetic structure using paired nucleotide and methylation genetic markers in the Sonoran bark beetle, Araptus attenuatus, for which we have a considerable knowledge about its neutral demographic history, demography, and factors influencing ongoing genetic connectivity.  Using the msAFLP protocol, 297 markers with variation in both nucleotide (NUC) and methylation (METH) were assayed from 20 populations collected throughout the species range.  Markers based upon methylation were both more frequent (16% vs. 7%), maintained more diversity (Shannon I<sub>Meth</sub> = 0.361 vs. I<sub>Nuc</sub>=0.272), and had more among-population genetic structure (Φ<sub>ST; Meth</sub> = 0.035 vs. Φ<sub>ST; Nuc</sub> = 0.008).  Interpopulation genetic distance in both NUC and METH markers were highly correlated, with 16% of the METH loci having sufficient signal to reconstruct phylogeographic history.  Allele frequency variation at five loci (two NUC and three METH) showed significant relationships with at-site bioclimatic variables suggesting subsequent analysis for local adaptation.  These results suggest that methylation can be as informative as nucleotide variation when examining spatial genetic structure for phylogeography, connectivity, and, identifying putatively adaptive genetic variance.
