## Introduction

Environmental heterogeneity may influence the distribution of genetic variation across natural populations by exerting a selective pressure on various parts of the genome.  The spatial distribution of this variation is, however, constrained by the species history, in that previous demographic perturbations determine the general distribution of total genetic variance.  Once neutral history is accounted for though, the remaining spatial genetic structure can provide insight into putative adaptive mechanisms in natural populations (Gavrilets & Vose 2005; Nosil et al. 2009). While putatively adaptive genetic and phenotypic variation shaping natural populations have been well demonstrated, the biological mechanisms involved in such adaptive processes have not been fully elucidated. Recent studies have suggested that epigenetic mechanisms may be one additional route through which an organism responds to environmental stressors; possibly conferring an increased adaptive potential on natural populations (Bossdorf et al. 2010; Richards 2011).  

Epigenetic factors and mechanisms influencing gene expression and heredity include DNA methylation, histone modification, small/micro RNAs; and other mechanisms that alter how DNA sequences are translated into functional gene products. To date, methylation of cytosine residues appears to be the most studied mechanism (Roberts & Gavery 2012).  In eukaryotes, methylation is a chemical modification that involves the addition of a methyl group onto the position 5 of a pyrimidine ring on the cytosines (5mC), primarily within the cytosine-phosphate-guanine (CpG) dinucleotides. DNA methylation can affect various functions in gene expression (Razin & Riggs 1980) ; it can change the structure of the chromosome, or, if present in the promoter region of the genome, restrict the access of transcription factors to the gene, effectively silencing the proximal gene (Klose & Bird 2006). In invertebrates, DNA methylation is thought to occur mostly in the coding regions of the genome (Roberts & Gavery 2012). DNA methylation, while varying widely between species, have been functionally linked to development, behavior, and phenotypic plasticity (Boyko et al. 2010; Day & Sweatt 2010; Law & Jacobsen 2010; Feng et al. 2010; Lyko & Maleszka 2011). For example, DNA methylation patterns in the honeybee brain – Apis mellifera have been associated with their life history; where workers progressing to different tasks (nursing the brood, to foraging outside the nest) with increasing age show varying patterns of methylated sites in their genome. In cases where these foragers revert back to nursing tasks, their methylation patterns have been shown to revert back to those characteristic of nest bees (Herb et al. 2012).  While the role of epigenetics in adaptive evolution is still unclear, it has been proposed to occur by contributing to phenotypic heterogeneity, and thus providing a substrate upon which selection is able to act (Flores et al. 2013). As such, it is of interest to evaluate utility of incorporating methylation markers in phylogeographic studies and to see how it aids in understanding and identifying putative adaptive variation in natural populations (Petren et al. 2005).  

Since methylation can be induced de novo, there are at least three categories of potential markers that can be uncovered when comparing to neutral sequence-based genetic variation.  Methylation loci that are induced each generation, or across a handful of generations, in response to environmental, developmental, or other localized conditions are expected to be uncorrelated with both population-level and spatial genetic structure based upon sequence-based markers.  Because they are induced repeatedly, these loci are, however, expected to be identifiable due to their correlation with quantifiable at-site conditions.  A second category of markers are those whose inheritance spans many generations, conferring assayable information about the recent demographic history of the species.  Inheritance across many generations would create spatial structure in these methylation markers that is correlated with that estimated from neutral sequence-based markers more commonly used in describing demographic history.  The final group of markers are uncorrelated with both neutral genetic and local heterogeneity.  It is possible that methylation may have a stochastic component or be due to processes other than demographic history or local adaptation.  

Assaying both sequence and methylation based genetic variation can be performed using methylation-sensitive AFLP (hereafter msAFLP) as introduced by Hill et al. (1996) and Reyna-López, Simpson, & Ruiz-Herrera (1997).  The msAFLP protocol produces both sequence and methylation variation, localized to a single nucleotide position in the genome, thereby controlling for intra-genomic heterogeneity.  This technique, uses two restriction isoschizomers, MSPI and HPAII, both of which target the same genetic sequence (5’-CCGG-3’).  Both enzymes cleave target DNA (denoted as +,+ for MSPI and HPAII respectively) when there is no methylation, though have different expression patterns in the presence of methylation depending upon where the methyl group is attached.  If the methyl group is attached to the internal cytosine residue (CmCGG), HPAII is prevented from cleaving the target sequence (+.-).  This same cutting pattern is also possible when the external cytosine is methylated but only on one strand (e.g.,  hemimethylated), though the frequency of hemimethylation is throught to be rather low and for the purposes of this work, both hemimethylated sites and methylation at the internal cytosine marker will be considered the same.  

when the   at the first cytosine (mCCGG), the second cytosine (CmCGG) or both cytosine residues (mCmCGG).  By Comparing cut patterns for homologous samples allows the identification of both the presence and the location of attached methylation.  

The MSPI enzyme cannot cleave the DNA if the The HPAII enzyme is sensitive to methylation and can only digest hemi-methylated sequences (when the methyl group is attached to the first cytosine, denoted as mCCGG). sequences from all possible methylated CCGG variants, along with non-methylated CCGG sequences. It should be noted that the MSPI while being relatively insensitive to methylation, is not able to cleave hemi and fully methylated mCCGG and mCmCGG sequences. Comparing the banding patterns between both MSPI and HPAII fragment profiles provides elucidation of mehtylation status for each fragment as summarized in Figure 1.

### Signatures of Selection

The evolutionary potential of a species and its ability to persist under changing environmental conditions is largely dependant upon the amount and spatial distribution of genetic variation. This in turn is shaped by a variety of factors; including neutral processes like gene flow, genetic drift, and mutation, and, non-neutral mechanisms like natural selection—both balancing and directional. Environmental heterogeneity has been shown to shape adaptive genetic variation in natural populations by exerting varying selective pressures across their genomes (Schoville et al. 2012). Thus, correlation between genetic or epigenetic variation with environmental gradients can be interpreted as evidence of natural selection (Eckert et al. 2010).
Balancing selection maintains different alleles at the selected loci; for example, at loci where there is a heterozygote advantage (overdominance) or, frequency-dependent selection with a rare-allele advantage (selection on sex ratio that maintains males and females in a population with an X/Y chromosome system; Charlesworth 2006). Genomic regions influenced by balancing selection show homogenized allele frequencies across populations, and an increased within population diversity.  Directional selection on the other hand tends to eliminate variation within populations, potentially leading to a isolation in natural populations due to differential performance in alternative environments. Directional selection is detected from the reduced levels of within population diversity and an increase in among population differentiation (Nielsen et al. 2005; Charlesworth 2006).

There are two general methods commonly used to identify putatively adaptive variation in non-model organisms based upon genome scans.  First, the background level of genetic divergence among populations is determined in large part by the demographic history of the populations being examined.  An outlier approach seeks to identify markers whose among population divergence is exceptionally high compared to this background level.  Presumably, genomic regions that are in the vicinity of items responding to divergent selection should show higher differentiation (Nosil et al. 2009). This method has been used repeatedly in many systems including intertidal snails (Littorina saxatillis) (Grahame & Wilding 2006), whitefish (Coregonus clupeaformis) (Campbell & Bernatchez 2004), and the common frog (Rana temporaria) (Bonin et al. 2006).  In general, the fraction of identified loci that could be considered for subsequent analyses are relatively small (average = 5%).   Another approach commonly used is that of gradient analyses—the so-called spatial analysis method (SAM) by Joost (2007).  This approach looks for systematic changes in allele frequencies along ecological gradients (Coop et al. 2010).  The rationale here is that the if environmental gradients are at least correlated with fitness related components of the genome, variation in genetic markers should mimic gradients in ecological features.  This approach does not require that the overall amount of differentiation is considerably larger than that created by demographic history, only that it changes systematically along with environmental gradients.  It should be pointed out though, both of these approaches only provide insights into genomic regions that may be linked to fitness related traits and subsequent analyses and experimentation (Holderegger & Wagner 2008) are required to show they are actually adaptive.

Anonymous loci (markers with unknown genetic information) are scanned to detect those under putatively adaptive selection, and, identified loci may either be directly under directional selection, or, linked to genomic regions under directional selection. However, this approach is not able to account for the confounding effects of historical and demographic processes that may cause substantial structure in natural populations and, interfere with the signals of adaptation detected from the outlier loci (Schlötterer 2003). Outlier loci tends to underestimate these demographical effects, thus increasing the number of false positives, especially in the case of natural populations diverged during glacial periods (Excoffier et al. 2009; Garrick et al. 2013). Under such circumstances, it is more suitable to perform an environmental association analysis; where the main goal is to test whether a specific allele is associated with a specific environmental factor, while controlling for neutral genetic structure (Rellstab et al. 2015). 


### Study Objectives

Our main objective in this study was to investigate the effectiveness of using epigenetic variation in informing demographic and evolutionary processes shaping natural populations.  *Araptus attenuatus* Wood (Circulionidae) is a bark beetle endemic to Baja California, known only to inhabit the senescing stems of the succulent Euphorbia lomelii.  The Baja California bioregion is characterized by steep environmental gradients known to have promoted population subdivision during post Pleistocene range expansion (Garrick et al. 2009, 2013).  While most species have exhibited a northward range expansion, this insect species has revealed a bi-directional range expansion, presumably due to a mid-peninsular warm desert refuge (Garrick et al. 2013). Previous studies have also demonstrated the influence of its host plant: Euphorbia lomelii on its demography. The strength of this influence has been attributed to the close relationship of the plant-insect pair. We compared markers derived from DNA sequence and methylation state using the msAFLP technique (Fulneček & Kovařík 2014) to examine the following two themes.   First, the amount and spatial distribution of genetic variation was estimated and compared using both marker sets to determine if methylation markers contain population genetic and phylogeographic signal that could be used to infer historical processes.  Since each sequence and methylation locus is paired, targeting the exact same location in the genome, direct comparison of basic structure statistics is not confounded by marker location.  The expectations for the relationship between methylation and sequence structure depend upon the processes inducing methylation.  If methylation variation is entirely neutral and is inherited as a Mendelian trait, then both sequence and methylation structure should share the same spatial-genetic structure.  If methylation is induced on a recent timescale, is stochastic, or is responding to environmental conditions at a different temporal (e.g., developmental conditions) or spatial (perhaps regional conditions) scale, it would be expected to have less spatial genetic structure than sequence variation as demographic history is not being recapitulated.  Conversely, if methylation variation is greater than sequence variation then methylation loci may be conferring information about non-neutral processes that sequence variation is not capturing (e.g., hidden variance).  Next, both sets of markers were examined for evidence of systematic variation with at-site bioclimatic conditions.  While a stronger correlation of epigenetic loci to environmental factors is intuitively indicative as being more useful than sequence based variation in uncovering adaptation in natural populations, we should bear in mind the complexities involved in the pathway connecting environmental variation, phenotypic plasticity, epigenetic, and genetic variation, and how it might affect inheritance underlying adaptation in a species. However, methylated loci may be a hidden repository of both neutral and putatively adaptive variation that we have yet to investigate fully.
